### PR TITLE
Updated version and links

### PR DESCRIPTION
**Pull request by vishr**
Signed-off-by: Vishal Rana <vr@labstack.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tunnel.rb [hunk 1]
```diff
@@ -19,15 +19,7 @@ def install
...
-      pid = fork do
-        $stdout.reopen("#{testpath}/out", "w")
-        exec bin/"tunnel", "8080"
-      end
-      sleep 5
-      assert_match "labstack.me", (testpath/"out").read
-    ensure
-      Process.kill("HUP", pid)
-    end
+    exec bin/"tunnel", "start", "8080"
```
Comment (lembacon) [inline Formula/tunnel.rb]: The following line will never get executed if you do not fork but `exec` directly in this case.

**Review by lembacon — CHANGES_REQUESTED** [review]

File: Formula/tunnel.rb (hunk 1 above)
Comment (vishr) [reply Formula/tunnel.rb]: This command itself forks within and comes out.

Comment (lembacon) [reply Formula/tunnel.rb]: As long as you call `exec`, the `ruby` process will be replaced by `tunnel`. Even though `tunnel` does `fork` another process and then “comes out”, that does not mean the process will return back to `ruby` and continue executing. Please see the `man` page for `exec(3)`, or the ruby doc for `Kernel.exec`.

Comment (vishr) [reply Formula/tunnel.rb]: I know how `exec` works and if I use fork, I have to put sleep and wait for tunnel to get started. This has been already test.

Comment (lembacon) [reply Formula/tunnel.rb]: I think `exec` without `fork`ing inside the `test do` block is like some sort of hijacking the `brew test` command. I'd like to let other maintainers to take a look at this. cc @MikeMcQuaid @Homebrew/maintainers

Comment (vishr) [reply Formula/tunnel.rb]: Sure, but like I said it is not blocking and passes the test.

Comment (MikeMcQuaid) [reply Formula/tunnel.rb]: Yes, please re-add the fork, thanks.

**Comment by vishr** [comment]
Updated as requested

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @vishr!

For future reference please don't sign-off your own commits and use the commit message (and PR title) format `tunnel 0.3.1` for updates. I also simplified the test in da9270c570f4d460f89b65cb73a7915cf3c75797.

**Pull request closed without merging**